src/App/Tracker/Controller/Hooks/ReceivePullsHook.php

```diff
@@ -61,11 +61,16 @@ protected function prepareResponse()
 		// If the item is already in the database, update it; else, insert it.
 		if ($this->checkIssueExists((int) $this->data->number))
 		{
-			$this->updateData();
+			$result = $this->updateData();
 		}
 		else
 		{
-			$this->insertData();
+			$result = $this->insertData();
+		}
+
+		if (!$result)
+		{
+			$this->getContainer()->get('app')->close();
```

Comment:
The close was something that was already done inside the respective functions. But yah I've now let things continue and set the error message to be the same as the log message